test/models/project_metadata_test.rb

```diff
@@ -0,0 +1,7 @@
+require 'test_helper'
```

Comment:
actually this file probably won't be used. its part of the default test scaffolding, but we're using rspec for testing